### PR TITLE
Added 'I love git' ascii art for git lov typo

### LINKED ISSUE #19: git lov
Idea: Love message (generated or fetched online) or ascii art

### PR DESCRIPTION
**Pull request by avanzzzi**
Closes #19

### CODE REVIEW

**Comment by Xaxetrov** [comment]
Thanks for your contribution ! I can have a look at it on sunday

File: shellGitTypo.sh [hunk 1]
```diff
@@ -137,6 +137,13 @@ gitTypoStatue="
 /                     (_
 "
 
+gitTypoHeart='
+╔══╗
+╚╗╔╝
+╔╝(¯`v´¯)
+╚══`.¸.[GIT]
+'
```
Comment (Xaxetrov) [inline shellGitTypo.sh]: I don't really get the heart. Where is it ?

File: shellGitTypo.sh [hunk 2]
```diff
@@ -282,6 +289,12 @@ git() {
   elif [ "$1" = "statue" ]; then
     echo "$gitTypoStatue"
     command git status "${@:2}"
+
+  # log
+  elif [ "$1" = "lov" ]; then
+    clear; echo "$gitTypoHeart"
```
Comment (Xaxetrov) [inline shellGitTypo.sh]: Please do not clear the shell if the ascii art is not dynamic. Instead just display the image like it is done for "git statue"

File: shellGitTypo.sh [hunk 3]
```diff
@@ -282,6 +289,12 @@ git() {
   elif [ "$1" = "statue" ]; then
     echo "$gitTypoStatue"
     command git status "${@:2}"
+
+  # log
```
Comment (Xaxetrov) [inline shellGitTypo.sh]: Please keep the easter eggs ordered alphabetically.

File: shellGitTypo.sh [hunk 4]
```diff
@@ -137,6 +137,13 @@ gitTypoStatue="
 /                     (_
 "
 
+gitTypoHeart='
```
Comment (Xaxetrov) [inline shellGitTypo.sh]: please keep ascii art constants ordered alphabetically.

**Review by Xaxetrov — CHANGES_REQUESTED** [review]

File: shellGitTypo.sh (hunk 1 above)
Comment (avanzzzi) [reply shellGitTypo.sh]: Changed to something that looks like a heart now.

File: shellGitTypo.sh [hunk 5]
```diff
@@ -87,6 +87,23 @@ gitTypoExplosion7='
      ) /
     ,---.'
     
+gitTypoHeart='
```
Comment (Xaxetrov) [inline shellGitTypo.sh:90]: Haha nice so big.

File: shellGitTypo.sh [hunk 6]
```diff
@@ -238,6 +255,12 @@ git() {
     echoRandomArrayElement gitTypoBible
     command git commit --amend "${@:3}"
 
+  # log
+  elif [ "$1" = "lov" ]; then
+    echo "$gitTypoHeart"
+    sleep 1.5
```
Comment (Xaxetrov) [inline shellGitTypo.sh:261]: Not sure about the sleep but we will keep it and I'll see later if I remove it or add it on other ascii arts.

**Review by Xaxetrov — APPROVED** [review]

**Pull request merged**